### DISCUSSION TITLE
Order of tweets

### DISCUSSION THREAD

**Discussion by gc-user** [j-fbriere/squawker#275] [Q&A]
First of all, thanks for making twitter usable for account-refusers again!

I noticed since tweets became viewable again that I could only see (very) old tweets. Now I realized that they might just be totally out of order, not sure. But I can't say for sure if there aren't also tweets missing when viewing the tweets of a twitter account.

I'm not able to say what order the tweets are following now, maybe by likes?
Is that scrambled sorting due to using a guest account?

Is there a way to force the old order of tweets, i.e. the latest at the top?

**Comment by j-fbriere** [comment]
Did you set a Twitter/X account?

**Comment by gc-user** [comment]
Nope. As I wrote, I'm an account-refuser... :-)

**Reply by gc-user** [reply]
I just realized that this scrambled order of posts also is on desktop when visiting a twitter account's page - of course without having an account, i.e. being logged-in. So, it seems to me that this is one more of twitters methods to annoy people into signing up for a twitter account...

On the other hand, using twstalker.com, the order of tweets on a account's page is correct. So, apparently, it might be possible to correct the order of tweets. Not that I would know how...

**Comment by Piscium** [comment]
UPDATE: I was wrong in what I wrote previously. I thought I was logged on while I was not. I am not sure what happened but I installed and uninstalled squawker several times, and of course Twitter keeps changing furiously.

Once logged on I see recent posts on top so all good for me (although I would have preferred not to have had the need to create an account). I am going to remove my old post to avoid confusions.
